### ISSUE TITLE
Fingering of piano chords is too widely spaced vertically

### ISSUE THREAD

**Issue by grigorax** [musescore/MuseScore#25875]
### Your idea

I cannot see any instruction in the MU4 manual, but the way I have been entering the fingering for chords in my piano music is to 
- select a note
- press a shortcut to apply fingering
- type in the two or more finger-numbers in a vertical column, with the numbers separated by line feeds.
- press space one or more times to advance to the next chord or note to be fingered.
However, the vertical spacing between the numbers is **too large.**
Here is an example from my current score:
 <img width="712" alt="Musescore - piano chord fingering" src="https://github.com/user-attachments/assets/6d8c5abc-cca9-4dab-b0a4-7bb7b48be8cb" />
Here is an extract of a printed score published by Schirmer, 1894:
[Doc - 18 Dec 2024 - 3-42 pm.pdf](https://github.com/user-attachments/files/18175209/Doc.-.18.Dec.2024.-.3-42.pm.pdf)

Note: I am aware that this can be done by means of the Properties tab. However this is an unwieldy solution.

### Problem to be solved

(1) Adjust the MU default spacing
(2) In addition, have a Style for Piano Fingering

### Additional context

I have checked with other piano scores that I have, and they are all similar to the Schirmer.

Having non-standard notation makes it hard for the performer, who has spent their musical life reading music in the standard style.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by MarcSabatella** [comment]
That's not the right way to enter fingerings.  Instead of a single fingering element with embedded line feeds, you should have a separate fingering for each note.  You can enter this easily by pressing Space to move from note to note while entering the fingerings.  This will space things better, and also associate the correct fingering with the correct note in general, which could be important for export or for later editing.

See the Handbook section on fingering in the Text chapter for more info.

**Comment by grigorax** [comment]
Thanks Marc that’s fantastic.  I never knew that 😆.    Makes sense.

And I see that SH-space works too. And if I move the note to create an inversion, the fingering moves with it (which doesn’t make much sense in this context).  Well done!

I did follow the link to the Text chapter but I saw nothing in there about fingering. And at any rate this is not a ‘formatting text’ issue.  The guidance surely should be in the Fingering section.  All it says is ‘to move the cursor to the next note, press space’. This does not cover it. Next, the video supplied does not treat chord fingering or skipped notes. It is very very basic!!

I looked for Format→Style→Text styles as suggested but didn’t find that path.
Whoops I just got a Musescore crash when I did the fingering shortcut on a note …

I think the Manual needs more work here.

Note that the ‘next’ note is assumed to be the next lower note of the chord. This needs to be stated.  It could just as logically be defined as the next higher note.

To be more complete, the Fingering feature for piano could also cover

  *   Changing to another finger while the note is sounding  (very common).
  *   Playing two notes with the same finger (typically the thumb) (this can also involve how the notes appear).

Violin.  I expect the same solution would apply to fingering of string instruments. There may be some wrinkles, I suppose.

Cheers
Greg




From: Marc Sabatella ***@***.***>
Date: Wednesday, 18 December 2024 at 4:28 PM
To: musescore/MuseScore ***@***.***>
Cc: grinffin25 ***@***.***>, Author ***@***.***>
Subject: Re: [musescore/MuseScore] Fingering of piano chords is too widely spaced vertically (Issue #25875)

That's not the right way to enter fingerings. Instead of a single fingering element with embedded line feeds, you should have a separate fingering for each note. You can enter this easily by pressing Space to move from note to note while entering the fingerings. This will space things better, and also associate the correct fingering with the correct note in general, which could be important for export or for later editing.

See the Handbook section on fingering in the Text chapter for more info.

**Comment by MarcSabatella** [comment]
The Fingering section of the handbook is within the Text chapter - that’s why I mentioned that’s, in case you didn’t know how to find it.  It does pretty clear say that Spasce and Shift+space move the cursor to the next and previous note - I’m not sure what else you feel would be necessary?  But if you have ideas for how it could be improved further, feel free to discuss them in the documentation forum over on MuseScore.com.  And I encourage you to use the support forum there if you have further questions on how to use MuseScore.

**Comment by grigorax** [comment]
Yes Marc I have read that part of the Fingering section. But to a pianist there is no concept of ‘next note’ being ‘next note in the current chord’. The word ‘next’ to him or her is a temporal concept not a spatial one although mostly their focus moves left to right on the page.

Upon reflection, I realise that the little video in the fingering section cannot cover chords because of its treating the general case of any instrument. To cover the case of chords, there would need to be a separate video.

And Marc, I don’t remember you covering this aspect of fingering in your course😊.

PS  I will post a comment in the forum highlighting how chord fingering works because I believe there must be many people like me ignorant of the best way to do it in Musescore.
________________________________
